TeamCharles/initial-site
# 41 edit product form

## Status

READY
## Documentation

[X] I have fully documented features in this PR. Please put an `X` in the box to confirm.
## Migrations

YES
## Description

A few sentences describing the overall goals of the pull request's commits.

The goal was to allow for users to edit the product if need be through routes and a form.
## Related PRs

List related PRs against other branches:
## Resolves Issue Number

number [link]()
#41
## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.

``` sh
git pull --prune
git checkout 41-edit-product-form
rm -rf Migrations/ bangazon.db
bangazon migrations
```

Please note that the view will not reflect changes, however the database will reflect the changes to the product.
1. 
## Impacted Areas in Application

List general components of the application that this PR will affect:
- Most product views, and the product controller
